api/image/image_test.go

```diff
@@ -64,9 +76,27 @@ func TestSplit(t *testing.T) {
 		},
 		{
 			testName: "with digest",
-			value:    "nginx@12345",
+			value:    "nginx@sha256:12345",
 			name:     "nginx",
-			tag:      "@12345",
+			tag:      "@sha256:12345",
+		},
+		{
+			testName: "with tag and digest",
+			value:    "nginx:1.2.3@sha256:12345",
+			name:     "nginx",
+			tag:      ":1.2.3@sha256:12345",
+		},
+		{
+			testName: "with domain",
+			value:    "docker.io/nginx:1.2.3",
+			name:     "docker.io/nginx",
+			tag:      ":1.2.3",
```

Comment:
IMO it doesn't make sense to include the leading colon in the split. The tag is "1.2.3". Could you change that?